src/Form/ReviewFormType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Review;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ReviewFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('text', TextareaType::class)
+            ->add('rating', ChoiceType::class, [
+                'choices'  => [
+                    '1' => '1',
+                    '2' => '2',
+                    '3' => '3',
+                    '4' => '4',
+                    '5' => '5',
+                ], 
```

Comment:
I wonder if we can sugarcoat it. Try this:
'choices' => [1,2,3,4,5]

If this won't work, resolve this comment

Reply:
![1](https://user-images.githubusercontent.com/98180264/202436365-aa7d8dae-2ce4-4ae5-bc4f-1994e6c98b45.png)